### PR TITLE
readme: faq entries

### PR DESCRIPTION
**Pull request by ebidel**
Took a stab at some "why we're doing this" faq entries.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -131,8 +139,39 @@ Yes. Puppeteer runs Chromium in [headless mode](https://developers.google.com/we
...
+
+#### Q: How is this different than Chromeless?
+
+[Chromeless](https://github.com/graphcool/chromeless) and Puppeteer are similar projects.
+Both are Node libraries that provide a high-level JS APIs to control headless Chrome.
+
+Chromeless (31 Mb) is intended to work well with AWS Lambda to deploy parallel testing in a serverless setup.
+Under the hood, it uses [chrome-remote-interface](https://www.npmjs.com/package/chrome-remote-interface) to interface with the DevTools Protocol. In the future, it could use Puppeteer's API.
+
+Puppeteer is smaller in size (1.77 Mb, ignoring the bundled version of Chrome) and does not use
```
Comment (JoelEinbinder) [inline README.md]: I don't think we should advertise our filesize. It becomes irrelevant when you need to download chrome, and might be misleading.

File: README.md [hunk 2]
```diff
@@ -131,8 +139,39 @@ Yes. Puppeteer runs Chromium in [headless mode](https://developers.google.com/we
...
+
+#### Q: Why is the Chrome team building Puppeteer?
+
+The goals are the project are simple:
+
+- Provide a slim (1.7 Mb), canonical library that highlights the capabilities of the [DevTools Protocol](https://chromedevtools.github.io/devtools-protocol/).
+- Provide a reference implementation for similar testing libraries. Eventually, these
+other frameworks could adopt Puppeteer as their foundational layer.
+- Grow the adoption of headless/automated browser testing.
+- Help dogfood new DevTools features...and catch bugs!
```
Comment (JoelEinbinder) [inline README.md]: DevTools Protocol features?

**Review by JoelEinbinder — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (ebidel) [reply README.md]: done

File: README.md (hunk 1 above)
Comment (ebidel) [reply README.md]: SG, I'll just say "smaller" and leave out the sizes for both libs

**Pull request merged**